README.md

```diff
@@ -33,7 +33,9 @@ const Tus = require('@uppy/tus')
 
 const uppy = Uppy({ autoProceed: false })
   .use(Dashboard, { trigger: '#select-files' })
-  .use(GoogleDrive, { target: Dashboard, companionUrl: 'https://companion.uppy.io' })
+  .use(GoogleDrive, { target: Dashboard, companion
```

Comment:
The option name is actually `companionUrl`, we can’t change that here :) See https://uppy.io/docs/facebook/#Options

Reply:
Apologies, I'm not sure how this happened, I might of either accidentally pressed the Delete key. In the file text I was going for consistency, changing 'url' to 'URL' this might of gotten incorrectly deleted with the global search function (I'm still getting the hang of the platform!).